coldfront/core/project/templates/project/project_join_list.html

```diff
@@ -9,6 +9,46 @@
 
 {% block content %}
 <h1>Join a Project</h1><hr>
+<h5>Pending Join Requests</h5>
+  {% if user.userprofile.access_agreement_signed_date is None %}
+    <p>First Review and Sign the cluster user agreement. Only then you can Join a cluster project and gain access to the cluster.</p>
+  {% elif join_requests is None %}
+    <div class="alert alert-info" role="alert">
+      <i class="fa fa-info-circle" aria-hidden="true"></i>
+        No pending requests.
+    </div>
+  {% else %}
+    <table class="table">
+      <thead>
+        <tr>
+          <th scope="col" class="text-nowrap">ID</th>
+          <th scope="col" class="text-nowrap">Name</th>
+          <th scope="col" class="text-nowrap">PIs</th>
+          <th scope="col">Title</th>
+          <th scope="col">Cluster</th>
+          <th scope="col" class="text-nowrap">Status</th>
+        </tr>
+      </thead>
+      <tbody>
+        {% for project in join_requests %}
+          <tr>
+            <td><a href="/project/{{project.id}}/">{{ project.id }}</a></td>
+            <td>{{ project.name }}</td>
+            <td>{% for pi in project.pis %}{{ pi.username }}<br>{% endfor %}</td>
+            <td style="text-align: justify; text-justify: inter-word;">{{ project.title }}</td>
+            <td>{{ project.cluster_name }}</td>
+
+            <td>
+                <span class="badge badge-success">
+                  {{project.auto_approval_time}}
+                </span>
+            </td>
+          </tr>
+        {% endfor %}
+      </tbody>
+    </table>
+{% endif %}
```

Comment:
Let's add some spacing between this and the next section, since it's all pretty close together right now.

Let's also add a header to the next section: "Make a New Request". (This might change.)